roles/kubernetes/node/templates/kubelet.rkt.service.j2

```diff
@@ -0,0 +1,58 @@
+[Unit]
+Description=Kubernetes Kubelet Server
+Documentation=https://github.com/GoogleCloudPlatform/kubernetes
+{% if kube_network_plugin is defined and kube_network_plugin == "calico" %}
+After=calico-node.service
+Wants=network.target calico-node.service
+{% else %}
+Wants=network.target
+{% endif %}
+
+[Service]
+Restart=on-failure
+RestartSec=10s
+TimeoutStartSec=0
+LimitNOFILE=40000
+
+ExecStartPre=-{{ rkt_bin_dir | default("/usr/bin") }}/rkt rm --uuid-file=/var/run/kubelet.uuid
```

Comment:
do we really want another bin dir path? Why the `bin_dir` can't serve on that purpose?